README.md

```diff
@@ -23,6 +23,15 @@ This module is available on the [Puppet Forge](https://forge.puppetlabs.com/pupp
 puppet module install puppetlabs-dsc
 ~~~
 
+**NOTE**
+
+When installing the module on Windows you may run into an issue regarding long file names (LFN), this is due to the generated schema files. If you are running in a master environment and installing the module on your master this will not be an issue, however if you are running masterless then you may run into this during installation depending on the length of the username it is ran from.  One way to get around this is to download the `.tar.gz` and install using the file rather than directly from the forge. For more info see [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866)
```

Comment:
So would it be safer to say, if using < 3.8.2 in or > 4 and less than 4.2.2 you may get this issue otherwise it has been fixed?